cinema/models.py

```diff
@@ -85,9 +81,7 @@ def clean(self):
             (self.row, "row", "count_rows"),
             (self.seat, "seat", "count_seats_in_row"),
         ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
+            count_attrs = getattr(self.movie_session.cinema_hall, cinema_hall_attr_name)
```

Comment:
The `clean` method in the `Ticket` model is attempting to access `count_rows` and `count_seats_in_row` attributes on `CinemaHall`, but these attributes do not exist. You should use `rows` and `seats_in_row` instead, which are the correct attributes defined in the `CinemaHall` model.